habitat-lab/habitat/gui/text_drawer.py

```diff
@@ -63,8 +63,8 @@ def __init__(
         self._max_display_text_chars = max_display_text_chars
 
         # Glyphs we need to render everything
-        # Using 6 * 256 as a size of the GlyphCache, to fit larger font size
-        self._glyph_cache = text.GlyphCache(mn.Vector2i(6 * 256))
+        # Using 3 * 256 as a size of the GlyphCache, to fit larger font size
+        self._glyph_cache = text.GlyphCache(mn.Vector2i(3 * 256))
```

Comment:
Thanks!

Ideal for the GPU would be to stay with power-of-two sizes in at least one dimension. I think this could still fit as well, it has roughly the same area:

```suggestion
        self._glyph_cache = text.GlyphCache((1024, 512))
```

Reply:
Unfortunately, that isn't enough, using  (1024, 768)  instead. Still better that 6 * 256.

Thank you!